tests/trainer/test_trainer.py

```diff
@@ -642,12 +642,15 @@ def test_trainer_max_steps_accumulate_batches(tmpdir):
 @pytest.mark.parametrize(
     ["benchmark_", "deterministic", "expected"],
     [
-        (None, False, True),
+        (None, False, None),
         (None, True, False),
+        (None, None, None),
         (True, False, True),
         (True, True, True),
-        (False, True, False),
+        (True, None, True),
         (False, False, False),
+        (False, True, False),
+        (False, None, False),
     ],
 )
 def test_benchmark_option(benchmark_, deterministic, expected):
```

Comment:
we don't have a test where we set both values for`torch.backends.cudnn.benchmark`, do we?
The test below always has `original_val=True` if I'm not mistaken